build_modules/README.md

```diff
@@ -32,3 +32,71 @@ building your Dart code.
 
 [`build_web_compilers`]: https://pub.dartlang.org/packages/build_web_compilers
 [`build_vm_compilers`]: https://pub.dartlang.org/packages/build_vm_compilers
+
+## Module creation
+
+The overall process for emitting modules follows these steps:
+
+1. Emit a `.module.library` asset for every `.dart` library containing a
+   description of the directives (imports, exports, and parts), a list of the
```

Comment:
Should we also mention the config specific dependency information here? Or does that fall under `directives`?

Reply:
I think that falls under `directives`. I mention config specific imports in one of the later steps.